MapboxAndroidDemo/src/main/java/com/mapbox/mapboxandroiddemo/MainActivity.java

```diff
@@ -566,4 +590,59 @@ protected void onSaveInstanceState(Bundle outState) {
     super.onSaveInstanceState(outState);
     outState.putInt("CURRENT_CATEGORY", currentCategory);
   }
+
+  private void checkWhetherFirstOpenOfApp() {
```

Comment:
- naming method feels a bit strange
- method has multiple responsibilities, can be extracted in a separate class and multiple methods.